### ISSUE TITLE
Machine installer broken since 2.3.0

### ISSUE THREAD

**Issue by karlis-vitols** [RocketChat/Rocket.Chat.Electron#354]
## My Setup

 - __My OS name and version__: Windows 10 x64, Windows 7 x64
 - __My current app version__: 2.3.0

- [x] I have tested with the latest application version
- [x] I can simulate the issue easily

#### Current Behavior

Since version 2.3.0 the fix included in pull request [https://github.com/RocketChat/Rocket.Chat.Electron/pull/319](url)
is no longer active, since the **package.json** code now shows:
    "nsis": {
      "include": "build/win/installer.nsh",
      "oneClick": false,
      **"perMachine": false,**
      "allowElevation": true,
      "allowToChangeInstallationDirectory": true
    },
Basically, this breaks mass deployment (via SCCM, for example). Issue also appears in 2.4.0 and from what I can see will still appear in 2.5.0 
I could build my own version from the source code each time, but that is not optimal, of course.
#### Expected Behavior

It should look like this:
"nsis": {
      "include": "build/win/installer.nsh",
      "oneClick": false,
      **"perMachine": true,**
      "allowElevation": true,
      "allowToChangeInstallationDirectory": true
    },

**Comment by graywolf336** [comment]
That setting was changed so that the installer could selected whether or not to install it for everyone or just the user at the time of installation.

**Comment by karlis-vitols** [comment]
So basically it's either the choice of this, when installing manually or working mass deployment option, but in that case it always installs for machine, not specific user, if I am correct?

**Comment by alexbrazier** [comment]
@karlis-vitols The installer now gives you the option to install for all users or for current user. This also allows you to install without UAC/Admin.

If you want to install using SSCM could you try running the installer using the silent install switch and see if that resolves your issue?

```
RocketChat.exe /S
```

**Comment by alexbrazier** [comment]
To install for all users run (as admin):

```
RocketChat.exe /S /allusers
```

**Comment by karlis-vitols** [comment]
Hello!
I have tested the deployment with the addition of /allusers flag and it works as expected.
Closing this issue!
Also I can see that this has been added to the upcoming 2.5 release readme, thank you!
